to-json-schema.js

```diff
@@ -21,11 +21,30 @@ const typeMappings = {
   uuid: 'string',
 };
 
-const commonAttributesMapping = (avroDefinition, jsonSchema) => {
+const commonAttributesMapping = (avroDefinition, jsonSchema, isTopLevel) => {
   if (avroDefinition.doc) jsonSchema.description = avroDefinition.doc;
   if (avroDefinition.default !== undefined) jsonSchema.default = avroDefinition.default;
+
+  const fullyQualifiedName = getFullyQualifiedName(avroDefinition);
+  if (isTopLevel && fullyQualifiedName !== undefined) {
```

Comment:
```suggestion
  if (isTopLevel && !fullyQualifiedName) {
```

Reply:
I'll change that if you like, but I was just trying to be consistent with the style. Just above on line 26 we have:
`if (avroDefinition.default !== undefined) jsonSchema.default = avroDefinition.default;`

Reply:
oh, I think now I remember why it was `avroDefinition.default !== undefined`, because probably `avroDefinition.default` could be `null` which should be fine 🤔 sorry but can you change it back and add a proper code comment?